### DISCUSSION TITLE
Network screen too high

### DISCUSSION THREAD

**Discussion by daniel-alomar** [netalertx/NetAlertX#554] [Q&A]
The network screen is too high. To access the list of devices I have to go down and use the scroll wheel over the minimal tab that appears lower left..
![imatge](https://github.com/jokob-sk/Pi.Alert/assets/15876760/c22dd803-be73-476d-966b-71dda3562126)

**Comment by jokob-sk** [comment]
Thanks! Will be fixed in the next release and at least the tabs will always be displayed, no matter the screen height.

Please submit issues as an issue, not a discussion in the future - it's easier for me to track what's to fix - thank you 🙏

![image](https://github.com/jokob-sk/Pi.Alert/assets/96159884/76ede27e-d96c-4f87-a54a-a9664afc51b7)

**Reply by daniel-alomar** [reply]
Sorry, I though was some problem in my configuration. Should I move the discussion to the issue forum?

**Reply by jokob-sk** [reply]
All good, not necessary,but thanks for asking!
